specs/aws_metadata.table

```diff
@@ -0,0 +1,28 @@
+table_name("aws_metadata")
+description("access AWS metadata available to VM's running as EC2 instances in AWS")
+schema([
+    Column("ami_id", TEXT, "AMI id of EC2 instance"),
+    Column("ami_launch_index", INTEGER, "EC2 launche index"),
+    Column("ami_manifest_path", TEXT, "The path to the AMI manifest file in Amazon S3"),
+    Column("ancestor_ami_ids", TEXT, "The AMI IDs of any instances that were rebundled to create this AMI"),
+    Column("block_device_root", TEXT, "The virtual device that contains the root/boot file system"),
+    Column("hostname", TEXT, "EC2 hostname"),
+    Column("instance_action", TEXT, "Notifies the instance that it should reboot in preparation for bundling"),
+    Column("instance_id", TEXT, "AMI instance id"),
+    Column("instance_type", TEXT, "AMI instance type"),
```

Comment:
Same, and throughout the remaining column descriptions, does Amazon reference these attributes in relation to the AMI or to the EC2 instance?